src/i18n/Streami18n.ts

```diff
@@ -122,12 +126,41 @@ Dayjs.updateLocale('ru', {
 });
 
 const en_locale = {
-  weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split(
-    '_',
-  ),
-  months: 'January_February_March_April_May_June_July_August_September_October_November_December'.split(
-    '_',
-  ),
+  formats: {},
+  months: [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ],
+  relativeTime: {},
+  weekdays: [
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    'Saturday',
+  ],
+};
+
+type Options = {
+  DateTimeParser?: typeof Dayjs;
+  dayjsLocaleConfigForLanguage?: Partial<ILocale>;
+  debug?: boolean;
+  disableDateTimeTranslations?: boolean;
+  language?: string;
```

Comment:
do these types not need to be taken off of the `stream-chat` `TranslationLanguages` instead of `string`?